frontend/src/pages/CalendarPage/index.js

```diff
@@ -215,6 +228,11 @@ class CalendarPage extends React.Component {
               </Container>
             )}
           </Grid>
+          <SnackbarErrors
+            open={openSnackbar}
+            message={message}
+            severity={severity}
+            handleClose={this.closeSnackbar()} />
```

Comment:
This is what's causing the infinite setState() loop error, causing your CircleCI test to fail. If you change it to `() => this.closeSnackbar()` it will work. handleClose needs a function, but what you're passing in is the function call itself and not a function

Reply:
oh thanks I was wondering what was up

Reply:
can also just be `this.closeSnackbar`